app/templates/results.html

```diff
@@ -1,127 +1,123 @@
 
 {% if result_set.results %}
-<a class="screenreader-text" href="#main-content">
-  skip to results
-</a>
-{% with pages=result_set.available_pages, current_page=result_set.page_number %}
+  <a class="screenreader-text" href="#main-content">
```

Comment:
I think a lot of the changes in this file are just my text editor doing some automatic formatting stuff.